package.json

```diff
@@ -15,9 +15,9 @@
     "production-build": "yarn --frozen-lockfile && lerna run build --concurrency 3 --stream",
     "dev-build": "yarn && lerna run build",
     "link-aa-dev": "cd packages/amplify-app && ln -s \"$(pwd)/bin/amplify-app\" \"$(yarn global bin)/amplify-app-dev\" && cd -",
-    "rm-aa-dev-link": "rm -f \"$(yarn global bin)/amplify-app-dev\"",
+    "rm-aa-dev-link": "rimraf -f \"$(yarn global bin)/amplify-app-dev\"",
```

Comment:
yes, under certain circumstances, for example one of the tools are not global that you invoke within a node script, it can fail when called with `xyz`, but if you use `yarn xyz`, since yarn manipulates the path for the execution it will succeed, so as a best practice we have yarn for all the scripts to make it error work with locally installed tools.